ports/fmt/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "fmt",
-  "version-semver": "8.0.1",
+  "version-semver": "8.1.0",
```

```suggestion
  "version": "8.1.0",
```

sorry, this isn't your issue, but this should be changed; fmt doesn't use semantic versioning.